package.json

```diff
@@ -4,6 +4,7 @@
   "private": true,
   "dependencies": {
     "@the-collab-lab/shopping-list-utils": "^1.0.1",
+    "firebase": "^9.1.1",
```

Comment:
Great! You're using the Firebase latest version. 🎉

Reply:
Congratulations on your work....!!!